src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -213,7 +213,7 @@ export class MdSlideToggle implements AfterContentInit, ControlValueAccessor {
 
 
   /** TODO: internal */
-  _onDragStart() {
+  _onDragStart(event: Event) {

```

Comment:
- Why don't we remove the parameters from the HTML instead (This does not show any linter warnings / infos etc.)
- Or make it consistent with the `onDrag` function, by using `HammerInput` instead? (to get proper autocompletion)